app/screens/TalkDetailsScreen/TalkDetailsScreen.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { FC } from "react"
+import { observer } from "mobx-react-lite"
+import { ViewStyle, View, TextStyle, ImageStyle, Image } from "react-native"
+import { StackScreenProps } from "@react-navigation/stack"
+import { AppStackParamList } from "../../navigators"
+import { Text, Tag, IconButton, MIN_HEADER_HEIGHT, BoxShadow } from "../../components"
+import { colors, spacing } from "../../theme"
+import { openLinkInBrowser } from "../../utils/openLinkInBrowser"
+import { TalkDetailsHeader } from "./TalksDetailsHeader"
+import Animated, { useAnimatedScrollHandler, useSharedValue } from "react-native-reanimated"
+import { SafeAreaView, useSafeAreaInsets } from "react-native-safe-area-context"
+
+const talkBlob = require("../../../assets/images/talk-shape.png")
+const workshopBlob = require("../../../assets/images/workshop-shape.png")
+const workshopCurve = require("../../../assets/images/workshop-curve.png")
+const talkCurve = require("../../../assets/images/talk-curve.png")
+
+const title = "React Native essentials"
+const subtitle = "Hotel, Conference Room 1"
+
+export const TalkDetailsScreen: FC<StackScreenProps<AppStackParamList, "TalkDetails">> = observer(
+  function TalkDetailsScreen() {
+    const scrollY = useSharedValue(0)
+    const onPress = (url) => openLinkInBrowser(url)
+    const [headingHeight, setHeadingHeight] = React.useState(0)
+
+    const scrollHandler = useAnimatedScrollHandler({
+      onScroll: (event) => {
+        scrollY.value = event.contentOffset.y
+      },
+    })
+
+    const { bottom: paddingBottom } = useSafeAreaInsets()
+
+    // TODO: wire this up to the event type
+    const isWorkshop = true
+
+    return (
+      <SafeAreaView edges={["top", "bottom"]} style={$root}>
```

Comment:
Curious on why are you not using the the `Screen` component?

Reply:
Hmm I will check that, I had removed it at one point playing with different animation code so I might just have forgot to revert it back. Let me look